### PR TITLE
Alexa: get stats (max, min, avg, count) of todays results

### PR DESCRIPTION
**Pull request by natterstefan**
@philipheimboeck Check it out please. _But please do not merge yet_, I am thinking about adding some more. But review it until then anyway. Thank you.

### CODE REVIEW

File: src/alexa.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 ## Globals
 TYPE_KEY = "Type"
 RESULT_KEY = "Result"
+bitDivider = 1048567
+speedType = " Mbit/Sekunde"
```
Comment (philipheimboeck) [inline src/alexa.py]: Pylint always tells me to write constants in large caps ;)

File: src/speech_assets/IntentSchema.json [hunk 2]
```diff
@@ -19,7 +19,22 @@
       ]
     },
     {
-      "intent": "MeasureSpeedIntent"
+      "intent": "MeasureSpeedIntent",
+      "slots": [
```
Comment (philipheimboeck) [inline src/speech_assets/IntentSchema.json:23]: Why do we need slots in the measurements? Guess we always measure everything

File: src/speech_assets/SampleUtterances.txt [hunk 3]
```diff
@@ -3,16 +3,23 @@ CurrentSpeedIntent wie schnell ist meine {Type} geschwindigkeit
...
 
-TweetCurrentSpeed {answer} bitte twitter das
-TweetCurrentSpeed {answer} bitte schicke einen tweet raus
+TweetCurrentSpeed {Answer} bitte twitter das
+TweetCurrentSpeed {Answer} bitte schicke einen tweet raus
 
-MeasureSpeedIntent messe den download
-MeasureSpeedIntent messe die download geschwindigkeit
+MeasureSpeedIntent messe den {Type}
+MeasureSpeedIntent messe die {Type} geschwindigkeit
```
Comment (philipheimboeck) [inline src/speech_assets/SampleUtterances.txt:10]: Ah ok... That's why

File: src/templates.yaml [hunk 4]
```diff
@@ -39,4 +39,7 @@ tweet_noResult:  |
     Du hast kein Resultat oder es ist 0. Starte den Test erneut.
 
 measure: |
-    Ich starte eine Messung. Bitte frag mich in einer Minute nach dem Ergebnis.
\ No newline at end of file
+    Ich starte eine Messung. Bitte frag mich in einer Minute nach dem Ergebnis.
+
+stats_text:  |
+  Ich habe mir deine {{ type }} Resultate von heute angesehen. Mir liegen {{ count }} Resultate vor. Dein maximal Wert liegt bei {{ max }} und dein Minimum bei {{ min }}. Deine durchschnittliche Geschwindigkeit liegt heute bei {{ avg }}.
```
Comment (philipheimboeck) [inline src/templates.yaml:45]: "Geschwindigkeit" does not fit with the type ping

**Comment by philipheimboeck** [comment]
Nice idea 👍

File: src/templates.yaml (hunk 4 above)
Comment (natterstefan) [reply src/templates.yaml:45]: @philipheimboeck I fixed it and added a new `stats_text` variation for the `ping` case.

**Comment by natterstefan** [comment]
@philipheimboeck I agree with you. Let's use the underscore variant. Please review my work-in-progess again.

**Review by philipheimboeck — APPROVED** [review]

**Comment by natterstefan** [comment]
@philipheimboeck thx.

**Pull request merged**